performance_budget.routing.yml

```diff
@@ -0,0 +1,51 @@
+entity.performance_budget.collection:
+  path: 'admin/reports/performance-budget'
+  defaults:
+    _entity_list: 'performance_budget'
+    _title: 'Performance Budget'
+  requirements:
+    _permission: 'administer performance budget'
+
+entity.performance_budget.add_form:
+  path: 'admin/reports/performance-budget/add'
+  defaults:
+    _entity_form: 'performance_budget.add'
+    _title: 'Add Performance Budget'
+  requirements:
+    _entity_create_access: 'performance_budget'
+
+entity.performance_budget.canonical:
+  path: 'admin/reports/performance-budget/{performance_budget}'
+  defaults:
+    _controller: '\Drupal\performance_budget\Controller\PerformanceBudgetController::viewRuns'
+    _title_callback: '\Drupal\performance_budget\Controller\PerformanceBudgetController::title'
+  options:
+    parameters:
+      performance_budget:
+        type: performance_budget
+  requirements:
+    _permission: 'performance_budget.view'
```

Comment:
good catch... this is an issue in WPA too.  I'll leave a note over there indicating as such